spec/system/support_interface/view_vendor_api_requests_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.feature 'Vendor API Requests' do
+  include DfESignInHelpers
+
+  scenario 'API requests are logged' do
+    FeatureFlag.activate(:vendor_api_request_tracing)
+
+    given_i_am_a_support_user
+    and_some_applications_exist
+    and_vendor_api_requests_for_applications_have_been_made
+
+    when_i_visit_the_vendor_api_requests_page
+    then_i_see_the_api_request
+    and_i_see_the_status_of_the_request
+
+    when_i_click_on_details_of_the_request
+    then_i_see_the_request_and_response_info
+
+    when_i_search_for_a_specific_request_path
+    then_i_only_see_api_requests_filtered_by_the_search
+  end
+
+  def given_i_am_a_support_user
+    sign_in_as_support_user
+  end
+
+  def and_some_applications_exist
+    applications = create_list(:application_choice, 2)
+    @first_application_choice = applications.first
+    @last_application_choice = applications.last
+  end
+
+  def and_vendor_api_requests_for_applications_have_been_made
+    visit vendor_api_path(@first_application_choice)
```

Comment:
Can we add a test for the happy path here as well? An authenticated request that results in a 200, that we can filter via the provider filters.

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3009/commits/dd73730484c46cbcf741b859b7ee098de8789b9b#diff-780c12b896f5644b1bb63115000759feR20-R27